packages/components/file-card/stories/file-card.happo.js

```diff
@@ -1,30 +1,30 @@
-import { storiesOf } from '@open-wc/demoing-storybook';
-import '@tradeshift/elements';
-import '@tradeshift/elements.file-card';
-
-import { createHappoStories } from '../../../../.storybook-happo/utils';
-
-import { sizes, states } from '../src/utils';
-
-storiesOf('ts-file-card', module).add('test', () => {
-	const properties = {
-		rtl: { true: true },
-		removable: { true: true },
-		state: {
-			[states.DOWNLOAD]: states.DOWNLOAD,
-			[states.FAILED]: states.FAILED
-		},
-		size: sizes
-	};
-
-	const options = {
-		columns: 5,
-		persistent_props: {
-			fileObject: { name: 'fileName.pdf', size: 1234567890 },
-			errorMessage: 'Failed to upload',
-			state: states.DOWNLOAD
-		}
-	};
-
-	return createHappoStories('file-card', properties, '', options);
-});
+// import { storiesOf } from '@open-wc/demoing-storybook';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACommentedCode)